EZSwiftExtensionsTests/UIViewTests.swift

```diff
@@ -11,6 +11,51 @@ import XCTest
 
 class UIViewTests: XCTestCase {
 
+    let translation = CGFloat(20)
+
+    
+    override func setUp() {
+        super.setUp()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+    }
+    
+    func view() -> UIView {
+        return UIView(x: 0, y: 0, w: 200, h: 50)
+    }
+    
+    func testSetTranslation() {
+        
+        let view = self.view()
+        let aView = UIView(frame:view.frame)
+        view.addSubview(aView)
+        
+        let oldOrigin:CGPoint = CGPoint(x: aView.frame.minX, y:aView.frame.minY)
+        
+        aView.setTranslation(duration: 1, x: 0, y: translation, completion: nil)
+        
+        let newOrigin = CGPoint(x: aView.frame.minX, y:aView.frame.minY)
+        
+        XCTAssertEqual(oldOrigin.y + translation, newOrigin.y)
+
+    }
+    
+    func testBackToOrigin() {
+        
+        let view = self.view()
+        let aView = UIView(frame:view.frame)
+        view.addSubview(aView)
+        
+        let oldOrigin:CGPoint = CGPoint(x: aView.frame.minX, y:aView.frame.minY)
```

Comment:
I recommend to use code formatter.
My recommendation is [Swimat](https://github.com/Jintin/Swimat).